Sources/LambdaKit/MFMessageComposeViewController+LambdaKit.swift

```diff
@@ -57,16 +57,35 @@ extension MFMessageComposeViewController: MFMessageComposeViewControllerDelegate
         }
     }
 
+    private var completionAfterDismissal: Bool {
+        get {
+            return objc_getAssociatedObject(self, &associatedEventHandle) as? Bool ?? false
+        }
+
+        set {
+            objc_setAssociatedObject(self, &associatedEventHandle, newValue,
+                objc_AssociationPolicy.OBJC_ASSOCIATION_RETAIN_NONATOMIC)
+        }
+    }
+
     /// Creates an instance of MFMessageComposeViewController and sets the completion closure to be used
     /// instead of the delegate. This closure is an analog for the
     /// messageComposeViewController:didFinishWithResult: method.
     ///
-    /// - parameter completion: A closure analog to messageComposeViewController:didFinishWithResult:.
+    /// - parameter completionAfterDismissal: Whether to invoke the completion closure before or
+    ///                                       after the controller has been dismissed.
+    /// - parameter completion:               A closure analog to
+    ///                                       messageComposeViewController:didFinishWithResult:.
+    ///                                       Invoked immediately after dismissing the controller.
```

Comment:
Doesn't this depend on the newly added parameter?

Reply:
Oops yeah thank you. I updated the docstring